mozdownload/scraper.py

```diff
@@ -50,15 +46,21 @@
                            'win32': 'exe',
                            'win64': 'exe'}
 
-MULTI_LOCALE_APPLICATIONS = ('b2g', 'fennec')
-
-APPLICATION_TO_FTP_DIRECTORY = {'fennec': 'mobile'}
+PLATFORM_FRAGMENTS = {'android-api-9': r'android-arm',
+                      'android-api-11': r'android-arm',
+                      'android-x86': r'android-i386',
+                      'linux': r'linux-i686',
+                      'linux64': r'linux-x86_64',
+                      'mac': r'mac',
+                      'mac64': r'mac(64)?',
+                      'win32': r'win32',
+                      'win64': r'win64(-x86_64)?'}
 
 
 class Scraper(object):
     """Generic class to download an application from the Mozilla server"""
 
-    def __init__(self, destination, version, platform=None,
+    def __init__(self, destination=None, platform=None,
```

Comment:
Destination needs to be optional given that it will default to the current working directory.